### PR TITLE
Add delete now button in debug mode

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

- [ ] I added/updated tests.
- [ ] I ensured that the PR title is good enough for the changelog.
- [ ] I labeled the PR.
- [ ] I self-reviewed the PR.

### CODE REVIEW

**Comment by jkoenig134** [comment]
Simulator Screen Recording - iPhone 16 Pro Max - 2024-12-16 at 12 54 58

File: apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart [hunk 1]
```diff
@@ -85,6 +85,17 @@ Future<void> showDeleteProfileOrIdentityModal({
               retryFunction = () => session.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
               pageIndexNotifier.value = 3;
             },
+            deleteNow: () {
+              final minutesInDays = 0.25 / 1440;
```
Comment (Milena-Czierlinski) [inline apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart:89]: Shouldn't it be 1/1440? Or rather to understand it easier 1/(24*60)? And if you actually want it to be 15 seconds, the naming should be adjusted, in addition.

**Review by aince42 — APPROVED** [review]

File: apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart (hunk 1 above)
Comment (jkoenig134) [reply apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart:89]: Actually this is not called oneMinuteInDays (which would be 1/1440), it's called minutesInDays, the left part can definitely be less than one.

Comment (jkoenig134) [reply apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart:89]: please keep a look on the file that it's in.

Comment (jkoenig134) [reply apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_or_identity_modal.dart:89]: I completely disagree with you.

**Pull request merged**